test-requirements.txt

```diff
@@ -5,3 +5,4 @@ tox==3.7.0
 Werkzeug==0.14.1
 pytest-localserver==0.4.1
 pytest-cov==2.6.0
+aiocontextvars==0.2.1
```

Comment:
Please move this to `tox.ini`, and just for sanic targets. You want to add a line to the tox sanic target here: https://github.com/getsentry/sentry-python/blob/c359d695e6ae06ed1569ee04ba85af81446425e0/tox.ini#L23